languages/tree-sitter-stack-graphs-javascript/src/stack-graphs.tsg

```diff
@@ -1661,11 +1661,14 @@ inherit .containing_class_value
 
 
 
-(field_definition
-  property:(property_identifier)@property)@field_def {
-
+(field_definition)@field_def {
```

Comment:
First, we just always give field definitions before/after scopes, no matter what.